static/bidder-info/oraki.yaml

```diff
@@ -0,0 +1,18 @@
+endpoint: "https://eu1.oraki.io/pserver"
+maintainer:
+  email: "ben@oraki.io"
+capabilities:
+  site:
+    mediaTypes:
+    - banner
+    - video
+    - native
+  app:
+    mediaTypes:
+    - banner
+    - video
+    - native
+userSync:
+  redirect:
+    url: "https://sync.oraki.io/pbserver?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&redir={{.RedirectURL}}"
+    userMacro: "[UID]"
```

Comment:
redirect url is not working

```
https://sync.oraki.io/pbserver?gdpr=&gdpr_consent=&us_privacy=&gpp=&gpp_sid={{.GPPSID}}&redir=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Doraki%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%22%5BUID%5D%22
```
`sync.oraki.io` web server is down 

<img width="1589" alt="image" src="https://github.com/prebid/prebid-server/assets/24757781/7825a835-231b-402d-9575-777aed0f0564">